lib/jekyll-github-metadata/site_github_munger.rb

```diff
@@ -42,10 +42,14 @@ def drop
 
       # Set `site.url` and `site.baseurl` if unset.
       def add_url_and_baseurl_fallbacks!
+        Jekyll::GitHubMetadata.log :debug, "Adding url and base url fallbacks..."
```

Comment:
This log applies not just for the line immediately after it but the entire function `add_url_and_baseurl_fallbacks`, so it is clearer to have an empty line after it.

```suggestion
        Jekyll::GitHubMetadata.log :debug, "Adding URL and base URL fallbacks..."

```

Also maybe take this whole log line out? I don't know if this line actually tells you anything usefu,l as it doesn't tell you actual values. And if there was an error in one of the lines in the function, that would be clear without having to see a log that told you that you were entering function.

Reply:
The original issue has this as output:

```
GitHub Metadata: Adding url and base url fallbacks...
GitHub Metadata: Url is set to https://github.com
GitHub Metadata: baseurl is set to /pages/SwedbankPay/swedbank-pay-design-guide-jekyll-theme
```

But this would be acceptable and useful I think:

```
GitHub Metadata: Url is set to https://github.com
GitHub Metadata: baseurl is set to /pages/SwedbankPay/swedbank-pay-design-guide-jekyll-theme
```